src/consensus/addresses.h

```diff
@@ -110,20 +110,28 @@ Consensus::CoinbaseAddresses AddressSets = {
         },
     .deuteronomy =
         {
-            // Note: nobody has private keys for these
-            "lotus_16PSJMzwHUZRNKGRxj6M78GUriT4JE9JVHBJwsPDR",
-            "lotus_16PSJQWWnyZvpXASxfYPnMuZgXuJe9Mi4Ko9E6AJJ",
-            "lotus_16PSJPLfgb7mYuMsyveuCiiw2EefZTDRevVGKrhCz",
-            "lotus_16PSJM5L7t1FvXnXYMKLBw5rBUNDewQwSxAnXnhcC",
-            "lotus_16PSJQaE6rFjErEmzsYaL2dNq51RUn1CPsXFykDeo",
-            "lotus_16PSJMsa5viW6TmE4TpJo2Ln5H5Tqu3bAsfRHEzcz",
-            "lotus_16PSJQkFaM93LHjiZLT2N3sYi8Z5yoPuiCTK6LSFB",
-            "lotus_16PSJM1S7w42U5vfPMHYFj2cuHnhsdVdrdc7BGGBV",
-            "lotus_16PSJQfyDWDvCBayWdWFDkkq8dB1uLfHUc5mt24C5",
-            "lotus_16PSJNYs6oFDXedfTMKBmJanibWk5akJgj6Ly1bxc",
-            "lotus_16PSJNwovGUnVpdWJQDuzX2JkKX14QpRbPVXEf1SZ",
-            "lotus_16PSJQr1FEMMWu6cAW4cWbXSJEkVLVsxwtzoiTKNS",
-            "lotus_16PSJQhqxYBFLYmqkZqvs8cf6ydaaXRxJBCyHCffb",
+            "lotus_16PSJHaDBFwmD1HPVWv3zLo3dbXMZTnCgBRzqFusu",
+            "lotus_16PSJMKqnLMJYonvMdn4mVqMb3bG5C76spwKuW4DL",
+            "lotus_16PSJHedfWvFixJzv5pV7g3FY2Z62kT9NprSqfYxW",
+            "lotus_16PSJJeYibqH7K189RbonEcoR7djvw54HWBebPQLY",
+            "lotus_16PSJJPzQYbd5JCX8UpukirFGz9hmCWFjGAhByDip",
+            "lotus_16PSJLr6AKaWTgioeJbjUWFAdq7yVDG17PQ1WoFn1",
+            "lotus_16PSJLr6AKaWTgioeJbjUWFAdq7yVDG17PQ1WoFn1",
+            "lotus_16PSJMStv9sve3DfhDpiwUCa7RtqkyNBoS8RjFZSt",
+            "lotus_16PSJMStv9sve3DfhDpiwUCa7RtqkyNBoS8RjFZSt",
```

Comment:
address "...ZSt" should have 4 seat.
@schancel See telegram for explanation.

Reply:
I'm sorry but we're not doing that. That website has to be over-provisioned if it's costing that much. We will do two.